src/stories/clickable-list/clickable-list.stories.tsx

```diff
@@ -1,9 +1,11 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
```

Comment:
Why not using the newer syntax ? https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-require-imports.md

Reply:
We did try in the first commit but there is issue with svg files. We reverted and added a icon-svg.d.ts inside the stories folder in order to make it work.